R/RcppExports.R

```diff
@@ -0,0 +1,31 @@
+# Generated by using Rcpp::compileAttributes() -> do not edit by hand
```

Comment:
I think that it would be preferable, (if it is possible), to not include generated files (including those generated by Rcpp).

At the moment, the `configure` script of the package generates those files.

Reply:
✅